frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -56,6 +56,11 @@ type Props = {| ...OwnProps, ...StateProps |};
 
 type TabKey = "data" | "general" | "defaultConfig";
 
+type DataSourceSettingsStatus = {
+  appliedSuggestions: "YES" | "NO" | "NONE",
+  isJSONFormatValid: "YES" | "NO" | "BROKEN_FORMAT",
```

Comment:
The name of this variable (`isJSONFormatValid`) in combination with the enum values is not really clear in my opinion. `jsonFormat` is ambiguous, it could either mean the "inner structure" of the json (keys, values, types) which the first two values of the enum refer to I assume or it could mean the json specification itself which the last enum value refers to.
Maybe it's more understandable if you rename the last enum value to "BROKEN_JSON" to avoid the duplicate reference of "format" - what do you think?

Reply:
> Maybe it's more understandable if you rename the last enum value to "BROKEN_JSON" to avoid the duplicate reference of "format" - what do you think?

That sounds like a good suggestion to me :D